packages/components/app/styles/components/tabs.scss

```diff
@@ -1,9 +1,69 @@
 //
 // TABS
 //
-// properties within each class are sorted alphabetically
-//
 
-.hds-tabs {
-  // outline: 2px dotted red;
+$indicator-height: 3px;
+
+.hds-tabs {}
+
+// Sub-components:
+// TODO: MOVE!
+.hds-tab-list {
+  font-family: var(--token-typography-body-200-font-family);
+  font-weight: var(--token-typography-font-weight-medium);
+  padding-bottom: ($indicator-height / 2);
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
```

Comment:
Some of this may be moved and restructured as I break this out into separate components and when I add the overflow navigation since I wasn't going to work on that part until after I get the basics working